react/wbooks/src/app/components/Login/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { get, isArray } from 'lodash';
+import { object, objectOf } from 'prop-types';
+
+import imagePlaceholder from '../../assets/wolox-logo.png';
+import InputLabel from '../InputLabel';
+import { login } from '../../../services/User/service';
+import { Routes } from '../../../constants';
+
+import styles from './styles.module.scss';
+import { LOGIN, SIGN_UP, FIELDS, FIELDS_DATA } from './constants';
+
+class Login extends Component {
+  state = { email: '', password: '', isError: false, errorMessages: [] };
+
+  handleLogin = async event => {
+    event.preventDefault();
+    try {
+      const response = await login(this.state);
+      console.log(response);
+      this.setState({ isError: false, errorMessages: [] });
+    } catch (error) {
+      this.setState({ isError: true, errorMessages: error.data.error });
+    }
+  };
+
+  handleSignUp = () => this.props.history.push(Routes.SIGN_UP);
+
+  onChangeField = (fieldName, fieldValue) => {
+    this.setState({ [fieldName]: fieldValue });
+  };
+
+  render() {
+    const errorMessages = isArray(this.state.errorMessages)
```

Comment:
build your own `isArray` function, is easy, and place it in a global `utils/helpers.js` folder.

Reply:
Why I should build it by myself? I can take advantage of the one already made and not do it again. Also I know is extremely well tested